test/python/scripts/tf_test_modify_input.py

```diff
@@ -163,7 +163,7 @@ def read_tests_from_file(filename):
                       required=True)
 
   args = parser.parse_args()
-  assert os.path.exists(args.input_dir), "Could not find the path"
+  if not args.input_dir: raise AssertionError("Could not find the path")
```

Comment:
args.input_dir is not of logical/boolean type, so the condition needs revision. "if not os.path.exists(args.input_dir)" could be used instead.

Reply:
Thanks for the comments. Updated it